alpha/hal.cpp

```diff
@@ -36,6 +36,9 @@ void _TS_HAL::begin()
   M5.Lcd.fillScreen(BLACK);
   M5.Lcd.setTextSize(1);
 
+  // enable to measure battery level
+  M5.Axp.EnableCoulombcounter();
```

Comment:
if this is required to measure battery level , maybe we just need to enable it right before we measure, and disable it?